theme-dev/README.md

```diff
@@ -1,33 +1,295 @@
+| ic_sidebar_notications_on_dark_24dp | color of the notification icon in the sidebar when you have the channel isn't muted |
+| ic_thread | color of the threads icon in the sidebar |
+
+<br />
+<br />
+
+# Material You
+
+> Material You is a new project by Google that tries to unify android themeing and make one global style that applies to all apps. 
+It was added in Android 12, so you will need Android 12. [Learn more](https://material.io/blog/announcing-material-you)
+
+> The themer plugin has support for it, allowing you to make themes that adapt to your system. 
+
+> This requires directly modifying the JSON file, so you will have to familiarise yourself with JSON.
+
+
+### Getting Started
+
+You are advised to [install this app](https://play.google.com/store/apps/details?id=com.ch3d.material.color), which will allow you to see all colors generated by Material You.
+
+Open the Themer Plugin menu and create a theme, this creates the JSON file that you will use for adding Material You values. 
+In the Themer plugin add random colors to all the UI Elements you'd like to change (this saves time later in the process). 
+
+Now that you have that, open the JSON file in your preferred text editor then move on to the next section.
+##### Suggested editors
+- [Vscode](https://code.visualstudio.com/) (Desktop)
+- [MiXplorer Editor](https://mixplorer.com/) (Mobile)
+
+### Adding Material You Colors
+
+Material You colors are separated into 5 groups:
+- accent1
+- accent2
+- accent3
+- neutral1
+- neutral2
+
+This is where the app comes into play. Launch it and you will see all 5 groups and different numbers inside of those groups which are different shades of that group. 
+
+Every Material You color consists of a few components:
+`system_ + group + _ + shade`, so for example `system_accent1_300`
+
+Using the app, find colors you like, figure out the full name as described above and replace the number (something like -292992) with the string. 
+Note: Strings are wrapped into double quotes, so your theme should end up looking something like this:
+```json
+{
+    "colors": {
+        "some-color": "system_accent0_300"
+    } 
+}
+```
+
+More detailed explanation can be found [here](https://discord.com/channels/811255666990907402/868419532992172073/898303519394758706).
+
+[Download Example Theme](https://github.com/MrSpidercat/Matu/releases/download/Release/matu-dark.json)
+
+
+
+<br />
+<br />
+
+
+# Random Things
+### Advanced Resources
+
+* This link by Ven might help you if you know what you're doing.
+
+https://gist.github.com/Vendicated/7e8aa7b2512b8e38e041692cbf34acfa
+
+### Light Mode Theming
+
+* As Riyu has found out, a lot of things in Light mode are not themable, but ven did say he will add more light mode support. [Link to Riyu's light mode tests](https://ptb.discord.com/channels/811255666990907402/868419532992172073/902278346195476490)
+
+
+
+
+<br />
+<br />
 
-<h1 align="Center">Credits</h1>
```

Comment:
Add the credits back  <img src="https://cdn.discordapp.com/emojis/910252525444296774.png" width="32" height="32">